app/locales/languages.js

```diff
@@ -77,6 +78,7 @@ i18next.init({
     es: { label: 'Español', translation: es },
     fr: { label: 'Français', translation: fr },
     ht: { label: 'Kreyòl ayisyen', translation: ht },
+    id: { label: 'Indonesia', translation: id },
```

Comment:
Please verify if this is correct for the dropdown list

Reply:
id for Indonesian is correct

Reply:
Thanks